### PR TITLE
Added workflow for removing file from commit

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -569,6 +570,19 @@ Let's say that you just blindly committed changes with `git commit -a` and you'r
 (master)$ git rm --cached log.txt
 ```
 
+<a href="#i-want-to-remove-a-file-from-a-commit"></a>
+## I want to remove a file from a commit
+
+In order to remove a file from a commit, do the following:
+
+```sh
+$ git co HEAD~2 myfile
```
Comment (k88hudson) [inline README.md]: Personally I think it would be better to use the full `checkout` here

**Comment by k88hudson** [comment]
LGTM, one small suggestion

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: D'oh!

**Comment by RichardLitt** [comment]
Thanks!

**Pull request merged**